.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Build and Push Image
+
+on:  # yamllint disable-line rule:truthy
+  - push
```

Comment:
@Allda I know it was working this way in the `operator-pipelines-images`, but should we really build and push image on each push to repo?

Reply:
Yeah, why not? This allows users to build images and test them without manually building and pushing images to the registry.